build/generate-struct-arrays.js

```diff
@@ -10,14 +10,15 @@
 
 'use strict'; // eslint-disable-line strict
 
-require('flow-remove-types/register');
-
 const fs = require('fs');
+
+require = require('@std/esm')(module, {"cjs":true, "esm":"js"});
+
```

Comment:
Can you expand on why `flow-remove-types` and `@std/esm` don't play well together? It would be great if we could avoid generating the source to a separate `src_untyped` folder...

Reply:
Some context here: https://github.com/standard-things/esm/issues/119

Reply:
> It would be great if we could avoid generating the source to a separate src_untyped folder...

I agree, I'm really hoping we can avoid a build step for tests 🤞